src/modules/data/soundData.h

```diff
@@ -17,17 +19,24 @@ typedef struct SoundData {
   SoundDataReader* read;
   void* decoder;
   struct Blob* blob;
-  struct Blob** queue;
-  uint32_t buffers;
-  uint32_t head;
-  uint32_t tail;
+  ma_pcm_rb *ring;
```

Comment:
Could this be a `void*` (like decoder) to avoid patching miniaudio and 'exposing' an internal type?

Reply:
huh. Yes it can, and it doesn't even need casts since void* casts automatically... I think even in VS iirc? :S I'll commit and we'll see if it builds.